hypershift-operator/controllers/hostedcluster/internal/platform/openstack/openstack.go

```diff
@@ -251,24 +251,38 @@ func (a OpenStack) ReconcileCredentials(ctx context.Context, c client.Client, cr
 	}
 
 	// Sync CNCC secret
+	if err := a.reconcileOpenStackCredentialsSecret(ctx, c, createOrUpdate, hcluster, controlPlaneNamespace, "cloud-network-config-controller-creds"); err != nil {
+		return err
+	}
+	// Sync CSI driver secret
```

Comment:
nit: Not your change, but we should probably change this to

```suggestion
	// Sync Cinder CSI driver secret
```

Given the changes in the next commit.